frontend/src/test/java/com/javabeast/service/liveLandingPageNavigation.java

```diff
@@ -0,0 +1,279 @@
+package com.javabeast.service;
+
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+// navigation link tests. verification that buttons are linking to correct URL.
+public class liveLandingPageNavigation {
+
+    private WebDriver driver = new ChromeDriver();
+
+    // faq navigation - top nav bar
+    @Test
+    public void callFAQNav() throws InterruptedException {
+        driver.get("https://bike-tracker.com/"
```

Comment:
If you look at all your methods, they are doing the same thing. this could be extracted into a method on its own, then reused.

public void navigateToHome() {
//move boiler plate code in here
}